### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by sulaimanahmadi**
Add following features

1. Use media queries.
2. use Grid and Flexbox.
3. design like figma.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -238,13 +244,17 @@ <h2>
...
                     <input name="name" id="name" type="text" placeholder="Full Name" maxlength="30" required>
                     <div class="separator1"></div>
-                  </li>
-                  <li class="input1 email">
-                    <input  name="email" id="email" type="email" placeholder="Email address" required>
+                  </li>  
+                  <li class="input1">
+                    <input name="name" id="name" type="text" placeholder="Last Name" maxlength="30" required>
                     <div class="separator1"></div>
-                  </li>
```
Comment (LionRouge1) [inline index.html:245]: - [ ] Great job, your design is identical to the Figma design :heart_eyes: However let's remove the input for Last Name and only use the Full name input (just like the mobile version) and make it takes the space of Last Name input (Have the same `width` with the Email input) to follow the project requirement (see image).
Screenshot from 2023-04-07 09-01-53

**Review by LionRouge1 — CHANGES_REQUESTED** [review]
Hi @sulaimanahmadi, 

Good job there :clap: !
There is only one important issue that you still need to work on to go to the next project but you are already there!

almost

## Highlights
- Your design is identical to the Figma design :1st_place_medal: 
- Your website is responsive from the desktop version to the Mobile version :+1: 
- Your mobile design still looks like expected :100: 
- All linters passed :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by sulaimanahmadi** [comment]
Hello dear sir

Actually you mean that, I should just use one input and it will be full
name same like email in size.

**Comment by LionRouge1** [comment]
@sulaimanahmadi, Yes It's in the [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md) I also attached the requirement image to my comment

File: desktop.css [hunk 2]
```diff
@@ -0,0 +1,449 @@
+/* stylelint-disable csstree/validator */
+/* stylelint-disable declaration-colon-newline-after */
```
Comment (didierganthier) [inline desktop.css:2]: [OPTIONAL] Nice work 💯 but avoid disabling the linters rules in the future, linting is a very beneficial thing. Benefits of linting. According to Ferit T., linting improves readability, 🔍  and removes silly errors before execution and code review. But, as mentioned, linting may do more complex jobs, like detecting code smells or performing static analysis of your codebase. 🤓

**Review by didierganthier — APPROVED** [review]
Hi @sulaimanahmadi ,

Congratulations! 🎉 Your project is complete and ready to be merged! 🚀



## Highlights
- Descriptive PR ✔️ 
- Linters passing ✔️ 
- Correct GitHub flow ✔️ 

## Optional Suggestions 💡
- While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really enhance your application.

Cheers and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Once again, congratulations on completing this project! 🎉

**Pull request merged**